nginx.conf.sample

```diff
@@ -242,6 +242,6 @@ gzip_types
 gzip_vary on;
 
 # Banned locations (only reached if the earlier PHP entry point regexes don't match)
-location ~* (\.php$|\.phtml$|\.htaccess$|\.git) {
+location ~* (\.php$|\.phtml$|\.ht[a-z]+$|\.git) {
```

Comment:
@ihor-sviziev: this will also match `.html`. Is that expected? Do we really want this?
I much more prefer `\.htaccess$|\.htpasswd$`, what's wrong with that?

Reply:
Hm, good point. Let's revert my last commit